ngraph/src/ngraph/op/util/binary_elementwise_comparison.hpp

```diff
@@ -55,7 +55,7 @@ namespace ngraph
             {
             protected:
                 /// \brief Constructs a binary elementwise comparison operation.
-                BinaryElementwiseComparison();
+                BinaryElementwiseComparison(const AutoBroadcastSpec& autob);
```

Comment:
Nitpick: don't think `autob` is a good name for variable, but from context (type name) it's clear what is it about..

Reply:
Agree, but it is a legacy name that is used all over ngraph operations with autobroadcasting. So lets keep it the same.